app/models/protocol_drug.rb

```diff
@@ -5,6 +5,15 @@ class ProtocolDrug < ApplicationRecord
   validates :name, presence: true
   validates :dosage, presence: true
 
+  enum drug_categories: {
```

Comment:
`drug_category` singular? Does plural work too?

Reply:
Shouldn't, fixing.

Reply:
Plural accidentally worked for what I was using it for 🤦 
`ProtocolDrug.drug_categories`